discord_quote/discord_quote/discord_quote.py

```diff
@@ -634,6 +651,72 @@ def pred(m):
                          'invalid_situation_request',
                           ctx.message.channel.name]))
 
+@bot.command()
+async def test(ctx):
+    # Function for debugging the current status of all the quote commands.
+
+    # --- Helper functions ---
+    async def get_last_real_message():
```

Comment:
Is there a particular reason for writing it this way, checking the condition with a counter?

Reply:
Oh oh oh wow I can't even read. Sorry.